### PR TITLE
Enable EIP-6110 in pytest

### PR DESCRIPTION
**Pull request by hwwhww**
Enable EIP-6110 in pytest and add CI jobs

TODO:
- [x] check test generators

p.s. GitHub Actions errors can be ignored now. CircleCI still covers all.

### CODE REVIEW

File: specs/_features/eip6110/light-client/full-node.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            gas_used=payload.gas_used,
+            timestamp=payload.timestamp,
+            extra_data=payload.extra_data,
+            base_fee_per_gas=payload.base_fee_per_gas,
+            block_hash=payload.block_hash,
+            transactions_root=hash_tree_root(payload.transactions),
+            withdrawals_root=hash_tree_root(payload.withdrawals),
+        )
+
+        # [New in Deneb]
```
Comment (hwwhww) [inline specs/_features/eip6110/light-client/full-node.md:50]: leftover
```suggestion
        # [New in EIP-6110]
```

File: specs/_features/eip6110/light-client/sync-protocol.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# Deneb Light Client -- Sync Protocol
```
Comment (hwwhww) [inline specs/_features/eip6110/light-client/sync-protocol.md:1]: leftover
```suggestion
# EIP-6110 Light Client -- Sync Protocol
```

File: specs/_features/eip6110/light-client/full-node.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Deneb Light Client -- Full Node
```
Comment (mkalinin) [inline specs/_features/eip6110/light-client/full-node.md:1]:
```suggestion
# EIP6110 Light Client -- Full Node
```

File: specs/_features/eip6110/light-client/full-node.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+- [Introduction](#introduction)
+- [Helper functions](#helper-functions)
+  - [Modified `block_to_light_client_header`](#modified-block_to_light_client_header)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This upgrade adds information about the execution payload to light client data as part of the Deneb upgrade.
```
Comment (mkalinin) [inline specs/_features/eip6110/light-client/full-node.md:20]:
```suggestion
This upgrade adds information about the execution payload to light client data as part of the EIP6110 upgrade.
```

File: specs/_features/eip6110/light-client/fork.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
+# eip6110 Light Client -- Fork Logic
```
Comment (mkalinin) [inline specs/_features/eip6110/light-client/fork.md:1]:
```suggestion
# EIP-6110 Light Client -- Fork Logic
```

**Comment by mkalinin** [comment]
Test vectors for EIP-6110 produced by `sanity` and `operations` generators:
* [minimal.tar.gz](https://github.com/ethereum/consensus-specs/files/11172059/minimal.tar.gz)
* [mainnet.tar.gz](https://github.com/ethereum/consensus-specs/files/11172062/mainnet.tar.gz)

**Review by kldeacy6 — APPROVED** [review]

**Review by kldeacy6 — APPROVED** [review]

**Review by kldeacy6 — APPROVED** [review]

**Review by mkalinin — APPROVED** [review]
I think this is ready for the merge now. We can fix any issues if occur in a separate changes. Thanks a lot to @hwwhww for the outstanding help with these tests!

File: specs/_features/eip6110/light-client/full-node.md (hunk 1 above)
Comment (J1psy-HUSSLE) [reply specs/_features/eip6110/light-client/full-node.md]: []()

**Pull request merged**